contents/teams/error-tracking/objectives.mdx

```diff
@@ -1,27 +1,56 @@
-### Q3 2025 Objectives
-
-#### SDK improvements and language support
-* Mobile support
-   * React Native JS (& Hermes source maps)
-   * Native iOS & Android
-   * Stretch: Flutter
-* Add error tracking to 3 more server side SDKs
-* Complete `posthog-js` and `posthog-js-lite` merge to a monorepo
-  * Stretch: continue with [the megaissue](https://github.com/PostHog/posthog-js/issues/2067) as time permits
-
-#### Integrate with other products
-* Find issues that impact signup / activation / retention heavily (correlation query)
-* Use Data warehouse / Revenue analytics to order issues by revenue impact
-* Add session context to issue page (example: breadcrumbs)
-* AI: Add session summaries to issues
-
-#### Product experience
-* Gather more qualitative data
-  * Continue customer interviews
-  * Make sure existing customers remain happy
-* Make in-app searching more intuitive
-  * Add error tracking tab to taxonomic filter
-  * Add email to omnisearch
-  * Expose search via Max AI
-* Clearly call out issues with source maps and offer resolution steps
-* Support Docs team's goal to make error tracking docs amazing
\ No newline at end of file
+### Q4 2025 Objectives
+
+#### Go all in on AI (Driver: <TeamMember name="David Newell" />)
+* Implement issue autofixing
+  * Integrate with Tasks product for in-app experience
+  * Build terminal based experience within the CLI
+    * Add issue selection to MCP (e.g. by file, most impactful)
+    * Test generation to prevent regressions
+* Add exception capture auto instrumentation to the wizard
+* Integrations to capture exceptions for popular LLM / agent tools (e.g. Langchain)
+
+#### Ship mobile error tracking (Drivers: <TeamMember name="Oliver Browne" />, <TeamMember name="Manoel Aranda Neto" />)
+* [Sequencing TBD]
+* Support suppression rules via remote config
+* Add suppression of minified exceptions before cymbal processing
+* Build dashboard template including crash free sessions / users
+
+#### Research deep dive on customers (Driver: <TeamMember name="Cory Slater" />)
+* Understand push & pull factors with competitors
+* Understand the impression and usage of error tracking tools / workflows at AI-first companies
+* Decide whether customers see LLM issues as exceptions
+* Understand spend / contraction of existing customers
+* Every engineer to attend at least 2 customer interviews
+
+#### Increase trust in product (Driver: <TeamMember name="Hugues Pouillot" />)
+* Improve issue grouping
+  * Introduce a measure of grouping accuracy to base changes off
+  * Add fingerprint versioning to avoid duplicate issues when iterating
+  * Fix obvious mis-groupings based on minor stack trace variances
+* RFC for moving issues into ClickHouse to avoid dual DB querying
+* Reduce the number of stacktrace-less issues
```

Comment:
is that client side work? you can always infer the stack trace from the current thread if the exception does not have one, but synthetic stack traces tend not to be great, even worse for grouping since they are generated at a point in time
the stack trace should be marked as synthetic (UI should highlight that) so the user will be more suspicious about it

Reply:
Yeah, we do rely on synthetic exceptions but sometimes there are other reasons we don't show a stacktrace e.g. no frames resolved. Hugues had some great examples in the planning doc